bravado/http_client.py

```diff
@@ -8,7 +14,13 @@ class HttpClient(object):
     and perform HTTP calls to fulfill a Swagger operation.
     """
 
-    def request(self, request_params, operation=None, request_config=None):
+    def request(
+        self,
+        request_params,  # type: typing.MutableMapping[str, typing.Any]
+        operation=None,  # type: typing.Optional[Operation]
+        request_config=None,  # type: typing.Optional[RequestConfig]
+    ):
+        # type: (...) -> HttpFuture
```

Comment:
This should probably be `HttpFuture[T]`.

Reply:
yup